ospackage/man/saptune-note.5

```diff
@@ -186,13 +186,15 @@ The section "[login]" manipulates the behaviour of the systemd login manager.
 This section can \fBonly\fP contain the following option:
 .TP
 .BI UserTasksMax= STRING
+This option is only available on SLE12. In SLE15 the limit is removed from the systemd login manager and therefore the setting is no longer supported by saptune.
```

Comment:
@angelabriel  The setting of `UserTasksMax` is not supported by saptune or not needed anymore?

Reply:
As the TaskMax setting is removed from the logind manager the setting of UserTasksMax is no longer needed, as the system default value is sufficient (infinity). And it's no longer supported by saptune as the parameter TaskMax is no longer available.

But if the customer has override files or custom specific files in /etc/saptune/extra, which contain the old UserTasksMax setting, saptune will simply skip these settings without throwing an error. Only an info message will inform the customer.